mixins/localize-static-mixin.js

```diff
@@ -0,0 +1,28 @@
+import { LocalizeMixin } from './localize-mixin.js';
+
+const fallbackLang = 'en';
+
+export const LocalizeStaticMixin = superclass => class extends LocalizeMixin(superclass) {
+
+	static async getLocalizeResources(langs) {
+		let resolvedLang = fallbackLang;
+		const resolvedResources = Object.assign({}, this.resources[fallbackLang]);
+
+		langs.reverse().forEach((lang) => {
+			if (this.resources[lang]) {
+				resolvedLang = lang;
+				Object.assign(resolvedResources, this.resources[lang]);
+			}
+		});
+
+		return {
```

Comment:
I've adapted/copied the logic.  The merge is pessimistic, in the sense that it iterates and merges all the languages even if it didn't need to.  Might not be a big deal for small sets.  I think this code will get called for every instance of a component, and it seems like we could cache the resolved resources so we only do this work once.

Reply:
Agreed -- I created #232 to track that.